tests/ui/resolve/loops.sol

```diff
@@ -8,25 +8,33 @@ function funky() {
 
     for (;;) break;
     for (; i < 40; i++) continue;
-    for (; i++ < 50;) continue;
+    for (; i++ < 50; ) continue;
 
     // ---
     // TODO: `Error: Variable declarations can only be used inside blocks.`
```

Comment:
Can you remove these TODOs and just add the error annotations instead of fixing this file? I would like to keep it the same

Reply:
See CONTRIBUTING.md. You should use `//~^ ERROR:` on the line below, and then `//~| ERROR:` on the line below that.

Reply:
Thank you, I did that but now, the the old errors of unresolved symbol is no longer emitted in places where this variable decl error is emitted. So what might we do here?

Reply:
Ah right, the original fix was fine then